src/App.vue

```diff
@@ -1,8 +1,132 @@
 <script lang="ts" setup>
+const toggleDrawer = () => {
+  const drawer = document.querySelector('.nav-links')
+  drawer?.classList.toggle('active')
+}
+
+const reloadPage = () => {
+  location.reload()
+}
+
+const scrollToElement = (id: string) => {
+  const drawer = document.querySelector('.nav-links')
+  drawer?.classList.toggle('active')
+
+  document.getElementById(id)?.scrollIntoView()
+}
 </script>
 
 <template>
+  <header h-12 md:h-20 lg:h-28 flex flex-row justify-between items-center gap-2 lg:gap-4 mx-0 class="header">
+    <button mx-4 md:mx-8 class="hamburger" :onclick="toggleDrawer">
+      <div class="bar" />
```

Comment:
```suggestion
```
Suggest removing this. A `div` should not appear in a `button`.
And since the style of `.bar` & `.hamburger` are same, you may not need it anymore.